src/components/AnnotationsPane.jsx

```diff
@@ -71,7 +73,7 @@ class AnnotationsPane extends React.Component {
         svgPoints.push(
           <circle
             key={svgPointPrefix+i}
-            cx={point.x} cy={point.y} r={10} fill="#3fc"
+            cx={point.x} cy={point.y} r={10} fill="#5cb85c"
```

Comment:
I went ahead and made these changes, but it really depends on how #64 is resolved.